packages/components/src/Loader/Loader.css

```diff
@@ -0,0 +1,21 @@
+.loader {
+  position: relative;
+  padding: var(--space-base);
```

Comment:
I don't think we want this padding here. It's collapsing the item it overlays which is going to cause layout jumping when the loading state changes:

![image](https://user-images.githubusercontent.com/39704901/127531153-9dde8c45-224d-4965-9fc9-78c55c280317.png)
![image](https://user-images.githubusercontent.com/39704901/127531183-8b0b6ea2-0f94-41ae-8efa-953a6108725f.png)

Reply:
Updated the padding here: ffa84d8

it was added for the progress bar, so it doesn't touch both sides, but have moved it so that it's margin in the loader indicator component